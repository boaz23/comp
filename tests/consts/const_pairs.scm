(define foo (lambda (x)
				(cons
					(begin (lambda () (set! x 1) 'void))
					(lambda () x))))
(define p (foo 2))

(cons ((cdr p)) (cons  ((car p)) (cons ((cdr p)) '())))