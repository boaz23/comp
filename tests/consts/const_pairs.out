(1 void 2)